doc/classes/FontVariation.xml

```diff
@@ -40,7 +48,7 @@
 			Base font used to create a variation. If not set, default [Theme] font is used.
 		</member>
 		<member name="fallbacks" type="Font[]" setter="set_fallbacks" getter="get_fallbacks" default="[]">
-			Array of fallback [Font]s. If not set [member base_font] fallback are ussed.
+			Array of fallback [Font]s. If not set [member base_font] fallback are used.
```

Comment:
This could be further improved:

```suggestion
			Array of fallback [Font]s to use as a substitute if a glyph is not found in this [FontVariation]. If not set, [member base_font]'s fallbacks are used instead.
```